### PR TITLE
Disable NETClientPrimitives test

### PR DESCRIPTION
**Pull request by BruceForstall**
Fails on many R2R and JitStress jobs.

Tracked by #19164

### CODE REVIEW

**Comment by BruceForstall** [comment]
@dotnet-bot test Windows_NT x64 Checked r2r
@dotnet-bot test Windows_NT x86 Checked r2r

**Comment by BruceForstall** [comment]
@dotnet-bot test Windows_NT x64 Checked r2r
@dotnet-bot test Windows_NT x86 Checked r2r

File: tests/issues.targets [hunk 1]
```diff
@@ -196,6 +196,9 @@
         <ExcludeList Include="$(XunitTestBinBase)\baseservices\varargs\varargsupport_r\varargsupport_r.cmd">
             <Issue>Varargs supported on this platform</Issue>
         </ExcludeList>
+        <ExcludeList Include="$(XunitTestBinBase)\Interop\COM\NETClients\Primitives\NETClientPrimitives\NETClientPrimitives.cmd">
```
Comment (AaronRobinsonMSFT) [inline tests/issues.targets]: @BruceForstall If this is the right place to disable then please remove the entry on line 439 - which was intended to disable this test when being run in helix - it consistently runs properly in the official coreclr CI.

**Comment by BruceForstall** [comment]
@dotnet-bot test Windows_NT x64 Checked r2r
@dotnet-bot test Windows_NT x86 Checked r2r

**Pull request merged**